cms/envs/aws.py

```diff
@@ -282,7 +283,8 @@
         DJFS['url_root'] = DJFS['url_root'].format(platform_revision=EDX_PLATFORM_REVISION)
 
 EMAIL_HOST_USER = AUTH_TOKENS.get('EMAIL_HOST_USER', EMAIL_HOST_USER)
-EMAIL_HOST_PASSWORD = AUTH_TOKENS.get('EMAIL_HOST_PASSWORD', EMAIL_HOST_PASSWORD)
+# force_str ised to fix this issue - https://code.djangoproject.com/ticket/27131
```

Comment:
typo: `s/force_str ised to fix/force_str is used to fix`

Reply:
sorry. fixed.